gridscale/datasource_gridscale_isoimage_test.go

```diff
@@ -0,0 +1,49 @@
+package gridscale
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccDataSourceISOImage_basic(t *testing.T) {
+	name := fmt.Sprintf("object-%s", acctest.RandString(10))
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+
+				Config: testAccCheckDataSourceGridscaleISOImageConfig_basic(name),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("data.gridscale_isoimage.foo", "id"),
+				),
+			},
+		},
+	})
+
+}
+
+func testAccCheckDataSourceGridscaleISOImageConfig_basic(name string) string {
+	return fmt.Sprintf(`
+resource "gridscale_isoimage" "foo" {
+  name   = "%s"
+  source_url = "http://tinycorelinux.net/10.x/x86/release/TinyCore-current.iso"
```

Comment:
:heart:

Reply:
What is wrong with this one please?

Reply:
Nothing :slightly_smiling_face: Just happy to see tiny core linux mentioned again